CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Marklogic Dependencies
 
+- With v1.1.5, Marklogic clinet code was regenerated against ML v1.12.0 dated 03/18/23
```

Comment:
I can't find the commit that reflects the regeneration. Could you give me the commit hash?

Reply:
I removed it as I did not run that code.